src/blas/backends/portblas/CMakeLists.txt

```diff
@@ -146,6 +146,10 @@ if (NOT PORTBLAS_FOUND)
   set(BLAS_ENABLE_BENCHMARK OFF)
   set(BLAS_ENABLE_TESTING OFF)
   set(ENABLE_EXPRESSION_TESTS OFF)
+  if(NOT PORTBLAS_TUNING_TARGET)
+    set(PORTBLAS_TUNING_TARGET "DEFAULT")
+  endif()
+  set(TUNING_TARGET ${PORTBLAS_TUNING_TARGET})
```

Comment:
Does it make sense to add a note that this variable will be used by portblas?

Reply:
Thank you for your review @mkrainiuk , I added a comment above it in b91f3863b5668210cfad33948c0a1870ddf4a5ea

Reply:
Thank you